src/posix.mak

```diff
@@ -97,7 +97,7 @@ MMD=-MMD -MF $(basename $@).deps
 
 # Default compiler flags for all source files
 CFLAGS := $(WARNINGS) \
-	-fno-exceptions -fno-rtti \
+	-fno-exceptions -fno-rtti -g -g3 \
```

Comment:
These should only be enabled in a debug build, and they already are. https://github.com/D-Programming-Language/dmd/pull/4321/files#diff-fc923c7ff79f8dd827793581118e03e6R110